package.json

```diff
@@ -247,6 +258,11 @@
           "command": "legend.show.diagram",
           "when": "viewItem == meta::pure::metamodel::diagram::Diagram",
           "group": "inline"
+        },
+        {
+          "command": "legend.edit.service.query",
+          "when": "viewItem == meta::legend::service::metamodel::Service",
```

Comment:
We need to support this for Functions too.  if not on this PR, on a subsequent one.